indico/htdocs/sass/modules/_papers.scss

```diff
@@ -41,3 +43,83 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        padding-left: 10px;
+        border-left: 3px solid $pastel-gray;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+        .row-title {
+            font-size: 14pt;
+            @include border-bottom();
+        }
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.review-role {
+                margin-right: 10px;
+                background-color: white;
+                color: $light-black;
+                border: 1px solid $gray;
+                i {
+                    color: $blue;
+                }
+            }
+
+            &.submitted {
```

Comment:
Consider extending `%semantic-background`. You would then need a state-class mapping in the template.

```jinja
{% set mapping = {'submitted': 'highlight',
                  'accepted': 'success',
                  ...} %}
```